test/controllers/matching.test.ts

```diff
@@ -29,29 +51,30 @@ describe('Matcher controller', () => {
   it('should find match', async () => {
     await createManySkills()
     await createEmployer()
-    await createJobPost()
-    const res2 = await createJobSeeker()
+    const employer = await loginEmployer()
+    await createJobPost() // this also should need token
+    const res2 = await newCreateJobSeeker()
+    expect(res2.status).toBe(200)
     const res5 = await request(app).get('/skills')
+    const seeker = await newJobSeekerLogin()
+    expect(seeker.status).toBe(200)
 
-    const seeker = await loginJobSeeker()
-    const employer = await loginEmployer()
-    console.log(employer.body)
+    const employer_token = employer.body.payload.token
+
+    const seeker_token = seeker.body.payload.token
 
-    const seekerToken = seeker.body.payload.token
-    const employerToken = employer.body.payload.token
     const res4 = await request(app)
       .post(`/employer/jobs`)
-      .set('Authorization', employerToken)
+      .set('Authorization', `Bearer ${employer_token}`)
       .send(jobPostForm)
 
     const response = await request(app)
       .get('/jobSeeker/match')
-      .set('Authorization', seekerToken)
+      .set('Authorization', `Bearer ${seeker_token}`)
 
     console.log(response.body.message)
     expect(res5.status).toBe(200)
-    expect(res2.body.status).toBe(201)
-    expect(res4.body.status).toBe(201)
+    expect(res4.status).toBe(200)
     expect(response.status).toBe(200)
```

Comment:
Can you add more matcher here, expect the result to exist. 200 status is good but we can test better